docs/index.md

```diff
@@ -0,0 +1,16 @@
+# Execution Spec Tests
+
+<figure markdown>
+ ![Execution Spec Tests Logo](img/execution_spec_tests.png){ width="200" align=right}
+</figure>
+
+
+A new set of tests for ethereum execution clients to test against. The [git repo](https://github.com/ethereum/execution-spec-tests) is updated regulary with new tests.
+
+## Getting Started
+- ###[Quick Start](getting_started/01_quick_start.md)
+- ###[Package Overview](getting_started/02_package_overview.md)
+- ###[Writing Tests](getting_started/03_writing_tests.md)
+
+## Tutorials
+- ###[Introduction](tuorials/01_)
```

Comment:
```suggestion
<h2>Getting Started</h2>
- <h3>[Quick Start](getting_started/01_quick_start.md)</h3>
- <h3>[Package Overview](getting_started/02_package_overview.md)</h3>
- <h3>[Writing Tests](getting_started/03_writing_tests.md)</h3>
<h2>Tutorials</h2>
- <h3>[Introduction](tutorials/01_state_transition)</h3>
```
This removes the permalinks within this page, which looked a bit confusing, and also removes themo from the navigation.